src/main/java/com/CezaryZal/manager/builder/TokenBuilder.java

```diff
@@ -0,0 +1,32 @@
+package com.CezaryZal.manager.builder;
+
+import com.CezaryZal.config.SecurityConstants;
+import com.CezaryZal.entity.User;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+
+@Service
+public class TokenBuilder {
+
+    public String buildTokenByUser(User foundUser){
+        long currentTimeMillis = System.currentTimeMillis();
+        String secret = SecurityConstants.SECRET_KEY;
```

Comment:
potrzebujemy to wynosić do zmiennej?

Reply:
teoretycznie nie, ale jeśli chcemy precyzyjnie określić długość życia tokenu (co do milisecundy) to mozna zostawić.

Reply:
Z dolnego problemu, zrobiłem jak zasugerowałeś